workflow.rst

```diff
@@ -8,8 +8,9 @@ best kept away from your models and should be defined in configuration.
 
 A **definition** of a workflow consist of places and actions to get from one
 place to another. The actions are called **transitions**. A workflow does also
-need to know each object's position in the workflow. That **marking store** writes
-to a property of the object to remember the current place.
+need to know each object's position in the workflow. This is the goal of the
+**marking store** that writes to a property of the object the current **place(s)**
```

Comment:
Technically speaking, the marking store could store the places **anywhere**: an API, a file, a table ...
I think we could replace `to a property` by `to a property or somewhere else`.

Reply:
understood